README.md

```diff
@@ -22,7 +22,33 @@ description here
 ## License 
 
 ## Process 
+### application folder system setup
++ set up the appropriate folder structure
++ added script files 
++ installed dependencies for the application
++ added docker compose files
++ added testing suite with dependencies
+
+### db\migrations 
+1. Created database schema
+
+### dbCongfig file
+2. Setting up  the DB configuration files 
+3. Seeding in some data to the DB to start dev process
+
+## Server functionality
+4. Implementation of author and posts models 
+5. Set up the server file
+6. Addition of the route files for both post and author
+7. Creation of controllers for author and post routes. Functionality:  get all, find by index for both objects and create post
+
+### Client 
+8. Creation of client side skeleton
+9. Establishes connection between server and client
+
+
 
 ## Bugs
+1. The app crashed - the cause was controller file exporting functions as an object. Fix to use appropriate module.exports with curly braces ```module.exports = { all };``` 
 
 ## Wins & Challenges
```

Comment:
Really clear README, that explains the process and the work you guys did. We were able to follow what you planned to do.